lib/check-http-method.js

```diff
@@ -6,7 +6,7 @@ const HTTP_METHODS = require('./http-methods');
  * @return {Boolean} Is correct or not
  */
 function checkHttpMethod(method) {
-    return RegExp(HTTP_METHODS.join('|'), 'i').test(method);
+    return RegExp(HTTP_METHODS.join('|')).test(method);
```

Comment:
Rationale: there is no request.GET (etc) methods.
As a result we can easily get a runtime error if we store uppercased versions of HTTP methods in http-status-codes.js.